ios/browser/api/web/ui/chrome_webview.mm

```diff
@@ -352,4 +354,22 @@ - (void)webController:(ChromeWebViewController*)controller
 
   [controller presentViewController:alert animated:YES completion:nil];
 }
+
+- (web::WebState*)webState:(web::WebState*)webState
+    createNewWebStateForURL:(const GURL&)URL
+                  openerURL:(const GURL&)openerURL
+            initiatedByUser:(BOOL)initiatedByUser {
+  SEL delegate_method = @selector
+      (chromeWebViewController:openNewWindowFor:openerURL:initiatedByUser:);
+  if ([_delegate respondsToSelector:delegate_method]) {
+    NSURL* nsURL = net::NSURLWithGURL(URL);
+    NSURL* nsOpenerURL = net::NSURLWithGURL(openerURL);
```

Comment:
Possibly want to either do `nil` checks before calling this delegate method or mark these as `nullable` in the protocol method definition above as its possible for `net::NSURLWithGURL` to return nil for GURL's that are invalid and Swift will crash if it finds nil